src/utils/apiSchema.ts

```diff
@@ -168,6 +174,8 @@ export interface paths {
           offset?: number;
           /** Center `latitude,longitude` in WGS84 decimal degrees. If provided, Locations are returned in order of increasing distance and the distance to each Location is returned. Longitude must be in the interval [-180, 180] and latitude in the interval [-90, 90]. */
           center?: number[];
+          /** Whether to include the path to a review photo thumbnail, if available. */
```

Comment:
Nice comment!

Reply:
Lol, this is generated from Ethan's Swagger documentation.